src/routers/task.router.ts

```diff
@@ -26,11 +31,33 @@ taskRouter.get("/", async (request, response) => {
 });
 
 taskRouter.get("/:taskId", async (request, response) => {
+
+    const taskById = await taskService.getTaskById(request.params.taskId);
     
-    response.send(await taskService.getTaskById(request.params.taskId));
+    // If authenticated user is a manager, add assignable users to response
+    const authenticatedUser: AuthenticatedUser = request.body['authenticatedUser'];
+    if(authenticatedUser.user.roles.includes("MANAGER")) {
+        const assignableUsers: UserData[] = [];
+        (await UserModel.find({ roles: { $elemMatch: {$in: "USER"} }})).forEach(user => {
```

Comment:
Is there a specific reason why you did this: { $elemMatch: {$in: "USER"}? And not just {roles: "USER"}

Reply:
Just not familiar with the syntax. Used what worked from stackoverflow. Will try to make it more readable